examples/cpp/Makefile

```diff
@@ -10,7 +10,7 @@ main.wasm: main.cpp
 	$(CXX) $(CXXFLAGS) -o $@ $^
 
 main_initialized.wasm: main.wasm
-	$(WIZER) --allow-wasi -r _start=wizer.resume -o $@ $^
+	$(WIZER) --allow-wasi --wasm-bulk-memory=true -r _start=wizer.resume -o $@ $^
```

Comment:
Should this become a default option now, since newer wasi-sdks use it? It's somewhat unintuitive to have to work out how to enable the feature otherwise.

Reply:
wasi-sdk doesn't have any LLVM patches, and upstream LLVM doesn't enable features by default until they're in all the major versions of all the major browsers that developers might be targeting.

Maybe wasi-sdk should start having LLVM patches so that it can do this though.

Reply:
@cfallin I think that would be great, especially since clang seems to unconditionally use it now.

@sunfishcode would that change allow us to start checking the wasi-sdk version directly?

Reply:
Are you asking if we could add a patch adding a wasi-sdk-specific predefined macro? It'd be technically possible, but I'm hesitant to do that because we do have a fair number of users that use non-wasi-sdk builds of clang with wasm, so it would risk creating awkward incompatibilities.

Reply:
Ah, I misinterpreted this comment:
> Maybe wasi-sdk should start having LLVM patches so that it can do this though.
Were you suggesting that we should patch LLVM to have better control over the features its assuming of the wasm runner?

Reply:
Oh, I think I misinterpreted the message above to be a request to enable bulk-memory in LLVM by default. That's the thing that would require us to carry patches. Confusingly, bulk-memory *is* used in wasi-libc's build, so wasi-sdk depends on it, even though LLVM itself doesn't enable it by default.

Reply:
Right, sorry, my original comment above was regarding wizer's defaults; i.e. support wizening modules that use bulk-memory extensions.

Reply:
Ah! I didn't realize that it was wasi-sdk using bulk-memory and not clang.

Reply:
As a follow-up is fine, just thought I'd suggest the long-term approach as opposed to the immediate fix here :-)